java-frontend/src/test/java/org/sonar/java/se/JavaCheckVerifier.java

```diff
@@ -207,9 +210,11 @@ private void assertMultipleIssue(Set<AnalyzerMessage> issues) throws AssertionEr
     Preconditions.checkState(!issues.isEmpty(), "At least one issue expected");
     List<Integer> unexpectedLines = Lists.newLinkedList();
     Multimap<Integer, Expectations.Issue> expected = expectations.issues;
+    flowAssertions = new SoftAssertions();
     for (AnalyzerMessage issue : issues) {
       validateIssue(expected, unexpectedLines, issue);
     }
+    flowAssertions.assertAll();
```

Comment:
If those blocker issues are FP (as I suspect) please create the ticket to handle them.